libdevcore/CommonData.h

```diff
@@ -42,22 +42,44 @@ enum class WhenError
 	Throw = 1,
 };
 
-enum class HexPrefix
+namespace
 {
-	DontAdd = 0,
-	Add = 1,
-};
-/// Convert a series of bytes to the corresponding string of hex duplets.
-/// @param _w specifies the width of the first of the elements. Defaults to two - enough to represent a byte.
+
+const char* hexdigits = "0123456789abcdef";
```

Comment:
Move this and `writeHex` into CommonData.cpp, anonymous namespace stuff shouldn't be in the header